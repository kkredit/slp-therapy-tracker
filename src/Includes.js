import React from 'react';

// May want the below

  // <script src="https://unpkg.com/react/umd/react.production.js" />
  // <script src="https://unpkg.com/react-dom/umd/react-dom.production.js" />
  // <script src="https://unpkg.com/react-bootstrap@next/dist/react-bootstrap.min.js" />
  // <script>var Alert = ReactBootstrap.Alert;</script>

class Includes extends React.Component {
  render() {
    return (
      <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
        crossOrigin="anonymous"
      />
    );
  }
};

export default Includes;
