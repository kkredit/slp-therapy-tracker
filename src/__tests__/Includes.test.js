import React from 'react';
import ReactDOM from 'react-dom';
import Includes from '../helpers/Includes.js';
import { create } from 'react-test-renderer'

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Includes />, div);
  ReactDOM.unmountComponentAtNode(div);
});

describe('Snapshots',()=>{
  test('Static page', () => {
    let tree = create(<Includes />)
    expect(tree.toJSON()).toMatchSnapshot();
  })
})
