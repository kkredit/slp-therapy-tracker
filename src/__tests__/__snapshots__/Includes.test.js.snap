// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots Static page 1`] = `
Array [
  <link
    crossOrigin="anonymous"
    href="https://stackpath.bootstrapcdn.com/bootswatch/4.3.1/yeti/bootstrap.min.css"
    integrity="sha384-w6tc0TXjTUnYHwVwGgnYyV12wbRoJQo9iMlC2KdkdmVvntGgzT9jvqNEF/uKaF4m"
    rel="stylesheet"
  />,
  <link
    crossOrigin="anonymous"
    href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
    integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN"
    rel="stylesheet"
  />,
]
`;
